tests/integration/run_tests.py

```diff
@@ -253,13 +253,14 @@ def usage(argv):
     print("-l <number of loops>")
     print("-a <test api >: run all tests of the specified API")
     print("-t <test_number>: run single test")
-    print("-d provides request also to the reference daemon (default RPCDAEMON)")
+    print("-d provides same request also to the reference daemon (default RPCDAEMON)")
     print("-i provides request also to the reference daemon (INFURA)")
     print("-b blockchain (default goerly)")
     print("-v verbose")
     print("-o dump response")
     print("-x exclude api list (i.e txpool_content,txpool_status")
     print("-X exclude test list (i.e 18,22")
+    print("-H IP address where the daemnon are localted(i.e 10.10.2.3) ")
```

Comment:
Replace with:

"-H host where the daemon is located (i.e 10.10.2.3)"

Reply:
ok